src/cube_overview.js

```diff
@@ -9,107 +9,131 @@ import CSRFForm from './components/CSRFForm';
-                  <a href={"/user/view/" + owner}> {owner}</a>
-                </h6>
+        <DynamicFlash />
+        <Row>
+          <Col md="4">
+            <Card className="mt-3">
+              <CardHeader>
+                <h3>{cube.name}</h3>
+                <h6 className="card-subtitle mb-2 text-muted">{cube.users_following.length}
+{' '}
+followers
+</h6>
+              </CardHeader>
+              <img className="card-img-top w-100" src={cube.image_uri} />
+              <em className="text-right p-1">
+                Art by:
+                {cube.image_artist}
               </em>
-              {admin &&
-                <CSRFForm method='POST' id='featuredForm' action={'/cube/' + (cube.isFeatured ? 'unfeature' : 'feature') + cube._id}>
-                    <Button color="success" type="submit"> {cube.isFeatured ? 'Remove from Featured' : 'Add to Featured'}</Button>
-                </CSRFForm>
-              }
-            </CardBody>
-            {this.state.followed ?
-              <Button outline color="danger" className="rounded-0" onClick={this.unfollow}>Unfollow</Button>
-            :
-              <Button color="success" className="rounded-0" onClick={this.follow}>Follow</Button>
-            }
-          </Card>
-        </Col>
-        <Col>
+              <CardBody>
+                {cube.type && (
+                  <>
+                    <a>
+                      {cube.card_count} Card{cube.type} Cube
+                    </a>
+                    <br />
+                  </>
+                )}
+                {!cube.privatePrices && (
+                  <>
+                    <a>Approx: $
+{price}
+</a>
+                    <br />
+                  </>
+                )}
+                <a href={`/cube/rss/${cube._id}`}>RSS</a>
+                <em>
+                  <h6>
+                    Designed by
+                    <a href={`/user/view/${owner}`}> {owner}</a>
+                  </h6>
+                </em>
+                {admin && (
+                  <CSRFForm
+                    method="POST"
+                    id="featuredForm"
+                    action={`/cube/${cube.isFeatured ? 'unfeature' : 'feature'}${cube._id}`}
+                  >
+                    <Button color="success" type="submit">
+                      {' '}
+                      {cube.isFeatured ? 'Remove from Featured' : 'Add to Featured'}
+                    </Button>
+                  </CSRFForm>
+                )}
+              </CardBody>
+              {loggedIn ? (
+                this.state.followed ? (
+                  <Button outline color="danger" className="rounded-0" onClick={this.unfollow}>
+                    Unfollow
+                  </Button>
+                ) : (
+                  <Button color="success" className="rounded-0" onClick={this.follow}>
+                    Follow
+                  </Button>
+                )
+              ) : (
+                []
+              )}
```

Comment:
this is the only meaningful change in this file, the rest is eslint going to town